pkc

```diff
@@ -268,7 +272,8 @@ run_setup() {
                         --wallet-address "$WALLET_ADDRESS" \
                         --pkc-server "$PKC_SERVER" \
                         --eauth-server "$EAUTH_SERVER" \
-                        ${empty_wiki:+--empty-wiki} || true
```

Comment:
The reason why the `|| true` is needed is for the case when `LocalSettings.php` already exists, and so the install.php exits with status 1. Adding `|| true` simply continues with the rest of the setup, i.e. seeding Eauth. Sorry that I haven't documented this. You can add a comment about this `|| true` in this PR.

Reply:
No problem, tbh thought it was do do with the variable substitution